Sources/Authenticator/Authenticator.swift

```diff
@@ -104,6 +112,9 @@ public struct Authenticator<LoadingContent: View,
         @ViewBuilder continueSignInWithMFASelectionContent: (ConfirmSignInWithCodeState) -> ContinueSignInWithMFASelectionContent = { state in
             ContinueSignInWithMFASelectionView(state: state)
         },
+        @ViewBuilder continueSignInWithTOTPSetupContent: (ConfirmSignInWithCodeState, String?) -> ContinueSignInWithTOTPSetupContent = { state, issuer in
```

Comment:
Having two arguments in the callback is not wrong per se, but it does diverge from all the other views, and it's also a bit unclear how to proceed.

Have you considered other options, such as having the `issuer` be a property of the state instead?